docs/guides/using-the-network-shim.md

```diff
@@ -0,0 +1,193 @@
+1. Commit the generated fixture files and push to the remote
+1. The test suite will now execute on CI and if a change breaks a test it will be impossible to merge
+
+To setup cross-version testing essentially means taking some of the steps above multiple times:
+
+-   Perform multiple capture runs against different versions of the live backend, so network fixtures are created for all these versions
+-   Define multiple jobs in the GitHub workflow that execute a stub run against these different versions
+
+## Configuration
+
+A workflow as described above consists of a few moving parts, which are described in more details below. **However, running the `d2-utils-cypress install` command should create all required files**.
+
+### Cypress
+
+#### `./cypress.env.json`
+
+The network shim expects a few environment variables to be present. These can be set using the `--env` flag when interacting with the cypress CLI, but base values should be provided as well the `./cypress.env.json` configuration file:
+
+```json
+{
+    "dhis2Username": "YOUR_USERNAME_HERE",
+    "dhis2Password": "YOUR PASSWORD HERE",
+    "dhis2BaseUrl": "http://localhost:8080",
+    "networkMode": "live",
+    "dhis2ApiVersion": "37"
+}
+```
+
+#### `./cypress.json`
+
+Interactive runs are not formally supported by the Network Shim, because `cypress open` (interactive mode) and `cypress run` have different behaviour regarding the plugin events. However, cypress is attempting to smooth things over and by setting `experimentalInteractiveRunEvents` to true it should theoretically be possible to at at least use stub mode in interactive mode, for example:
+
+```json
+{
+    "baseUrl": "APP URL",
+    "projectId": "PROJECT ID",
+    "testFiles": "TEST FILES",
+    "experimentalInteractiveRunEvents": true,
+    "video": false
+}
+```
+
+#### `./cypress/plugins/index.js`
+
+For most DHIS2 cypress setups, using the `enableAutoLogin` setup helps managing authentication conveniently. In turn, the `enableAutoLogin` setup utility requires the `chromeAllowXSiteCookies` when using Chrome. In most cases it makes sense to always use this plugin alongside the `networkShim`.
+
+The `networkShim` plugin itself doesn't require the `config` object, but it does accept a second `options` argument. For most DHIS2 platform apps the default options will be correct so there will be no need to pass a second argument.
+
+```js
+const {
+    networkShim,
+    chromeAllowXSiteCookies,
+} = require('@dhis2/cypress-plugins')
+
+/*
+ * const networkShimOptions = {
+ *     hosts: [
+ *         'https://main-api.com',
+ *         'https://secondary-api.com',
+ *     ],
+ *     staticResources: [
+ *         '/endpoint-which-always-return-the-same-data',
+ *         '/another-endpoint-which-always-return-the-same-data',
+ *     ]
+ * }
+ */
+
+module.exports = (on, config) => {
+    networkShim(on /*, networkShimOptions */)
+    chromeAllowXSiteCookies(on, config)
+}
+```
```

Comment:
This code block could use a little more context for me. It doesn't immediately follow from the paragraph above what the user is supposed to do with this code block. Also the commented out code seems confusing as it seems your demonstrating two usecases at once. I'd separate the simple one and the elaborate on, starting with the simple one that should apply in most cases.